test_unstructured/staging/test_label_studio.py

```diff
@@ -3,12 +3,57 @@
 
 from unstructured.documents.elements import Title, NarrativeText
 
+from label_studio_sdk.client import Client
+
+import logging
+import re
+import vcr
+
 
 @pytest.fixture
 def elements():
     return [Title(text="Title 1"), NarrativeText(text="Narrative 1")]
 
 
+@vcr.use_cassette("fixtures/cassettes/label_studio.yaml")
+def test_upload_label_studio_data_with_sdk_on_real_instance(caplog, elements):
+    # The HTTP request to label studio is recorded by vcr decorator in the first time
+    # and replayed later.
+    log = logging.getLogger("urllib3")
+    log.setLevel(logging.DEBUG)
+    # Define the URL where Label Studio is accessible and the API key for your user account
```

Comment:
I think this comment should be clarified. This sounds like it needs to match credentials of the current user for testing purposes. I think it should be clear that...
1. this is a fake key that needs to match what's in the vcr cassette and
2. this would need to by updated if the vcr cassette is updated (by someone with a different access token).